_includes/rest/queries.md

```diff
@@ -412,7 +412,7 @@ print result
 </code></pre>
 </div>
 
-You can restrict the fields returned by passing `keys` or `excludeKeys` a comma-separated list. To retrieve documents that contain only the `score` and `playerName` fields (and also special built-in fields such as `objectId`, `createdAt`, and `updatedAt`):
+You can restrict the fields returned by passing `keys` or `excludeKeys` as an [array](#arrays). To retrieve documents that contain only the `score` and `playerName` fields (and also special built-in fields such as `objectId`, `createdAt`, and `updatedAt`):
```

Comment:
Just want to note that I think it's good not mention that *both* will actually be possible after the PR, because we want to [phase out](https://github.com/parse-community/parse-server/issues/7245#issuecomment-791978838) the string variant.

Reply:
@mtrezza How do you suggest to address this? If the PR is merged, the use of arrays won't be applicable to Parse Server <= 4.5.0, only newer, they have to use comma delineated strings.

I can see a dev saying, "it doesn't work", because they are using an older version of the server.

Reply:
You are reading my comments faster than I can write them :-) Please see my update above.

Reply:
This works for me. Lets see if @TomWFox thinks we need to add anything more

Reply:
Sounds good to me 👍